.ci/scripts/update_ci_branches.py

```diff
@@ -0,0 +1,34 @@
+import os
+import re
+import sys
+import requests
+from packaging.version import Version
+from git import Repo
+
+try:
+    initial_branch = sys.argv[1]
+except IndexError:
+    initial_branch = None
+
+repo = Repo(os.getcwd())
+heads = repo.git.ls_remote("--heads", "https://github.com/pulp/galaxy_ng.git").split("\n")
```

Comment:
This looks strange `pulp/galaxy_ng` ? maybe there is a way to disable this script? or can we just ignore it? @fao89 ?

Reply:
it gets the value from here:
https://github.com/ansible/galaxy_ng/blob/master/template_config.yml#L45

Reply:
So, as we are not running that script we can ignore it right? we can't change `github_org` because it is used with `pulpcore_branch` to build pulpcore url